triple_product_vs_T/triple_product_vs_T.py

```diff
@@ -183,16 +186,19 @@ def get_experimental_dataset( dataset='Bosch', silent=True ):
                                ] )
     elif dataset == 'my_dset':
         names = np.array( ['START', 'Globus-M', 'NSTX', 'MAST', 
-                           'W7-X (lim)', 'W7-X (div)', 'ITER', 'TCV'
+                           'W7-X (lim)', 'W7-X (div)', 'ITER', 'TCV', 
+                           'EAST', 'EAST'
                           ] )
         device_types = np.array( [ 3, 3, 3, 3, 
-                                   1, 1, 2, 2
+                                   1, 1, 2, 2, 
+                                   2, 2
                                  ] )
         T_vals = np.array( [ 0.25, 0.5, 1., 0.9, 
-                             1.,   3.5, 20., 3.7
+                             1.,   3.5, 20., 3.7, 2.1, 1.94
```

Comment:
same as above (using only one EAST value here seems to be more reasonable)

Reply:
OK